gitgud/levels/reworking/_interactiverebase/test.spec

```diff
@@ -0,0 +1,9 @@
+1 (overHere)
+2 : 1
+3
+4
+5
```

Comment:
Since we're rebasing, we don't actually need the original commits listed since we're not going to be using them